appscale/tools/local_state.py

```diff
@@ -580,8 +580,16 @@ def get_username_from_stdin(cls, is_admin):
         username = raw_input('Enter your desired e-mail address: ')
 
       username = username.lstrip().rstrip()
-      email_regex = \
-        '^.+\\@(\\[?)[a-zA-Z0-9\\-\\.]+\\.([a-zA-Z]{2,3}|[0-9]{1,3})(\\]?)$'
+      email_regex = '^.+\\@(\\[?)[a-zA-Z0-9\\-\\.]+\\.' \
+                    '([a-zA-Z]{2,3}|[0-9]{1,3})(\\]?)$'
+
+      # If the username belongs to an admin, allow the domain token to be
+      # 'appscale'. The fast-start script assigns admin@local.appscale
+      # as the cloud admin.
+      if is_admin:
+        email_regex = '^.+\\@(\\[?)[a-zA-Z0-9\\-\\.]+\\.' \
+                      '([a-zA-Z]{2,3}|[0-9]{1,3}|appscale)(\\]?)$'
```

Comment:
With the vast number of valid TLDs with more than 3 characters, should that constraint be removed rather than just allowing 'appscale'?